docs/EVE-IMAGE-SOURCES.md

```diff
@@ -0,0 +1,616 @@
+# Source Tracing
+
+As an open source project, it is important for all of the sources for EVE to be open and available. While for some having this repository
+may suffice, EVE sets higher standards. Specifically, EVE desires that it be possible to trace from a given EVE image back to the
+sources of all software used in that binary compilation of EVE.
+
+This is done for several purposes:
+
+* Debugging, where knowledge of the precise source used is critical.
+* Auditing, where users auditing a particular distribution require confidence that they are reviewing precisely the correct source.
+* License compliance, where several licenses, notably the GPL family, require either distribution of source or links to source when distributing binary artifacts.
+
+This document will describe how to trace from a given compiled distribution of EVE all the way back to all sources.
+
+The stages are:
+
+1. Binary to source commit
+1. Binary to builder configuration
+1. Container images to container sources
+1. Container ources to contents:
```

Comment:
typo "ources"

Reply:
Never heard of "ources"? It is an old French variant on "bears". 😆